01-Login/src/router.js

```diff
@@ -0,0 +1,40 @@
+import Vue from "vue";
+import Router from "vue-router";
+import Home from "./views/Home.vue";
+import Profile from "./views/Profile.vue";
+import Callback from "./components/Callback.vue";
+import auth from "./auth/authService";
+
+Vue.use(Router);
+
+const router = new Router({
+  mode: "history",
+  base: process.env.BASE_URL,
```

Comment:
`01-Login` doesn't need a `.env` file as it's only used in sample 03 to provide auth configuration for the Express server, which the first sample doesn't have. The CI script creates it just because it has to cater for both samples as part of the CI run, it will just go unused by the first sample.

I can see the confusion but I also think it would be needless to ask the reader to provide configuration where it's not even needed (when reading the first sample). Does that make sense, or do you still feel it's worthwhile putting in?

Reply:
This has been replaced now with a JSON file that can be read by both the backend and frontend app.